go.mod

```diff
@@ -5,9 +5,12 @@ go 1.19
 require (
 	github.com/go-logr/logr v1.2.4
 	github.com/kserve/kserve v0.11.0
+	github.com/kuadrant/authorino v0.15.0
+	github.com/manifestival/manifestival v0.7.2
 	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.27.6
 	github.com/openshift/api v3.9.0+incompatible
+	github.com/pkg/errors v0.9.1
```

Comment:
Hi

any reason to make it direct dependency?

Reply:
https://github.com/opendatahub-io/odh-model-controller/pull/130/commits/5c1fdf8a99e58492a17508406d6e79e7875a22bc